projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 13.5-a.3 - 2024-05-20
+### Enhancements
+- AI Assistant: Enable inline Heading extension. [#37386]
+- Subscribe block: Adds button-only style. [#37341]
+- Subscribe block: Allow inside navigation block. [#37439]
+
+### Improved compatibility
+- Block Editor: Remove External Link icon styling fix now that the change has been made in WordPress itself. [#37394]
+
+### Bug fixes
+- Slideshow: Fix className issue on frontend - ensures autoplay works properly. [#37378]
```

Comment:
Nit: we could also fix it later when we do the actual release but just quickly flagging it. I think the first part of the sentence is somewhat unnecessary for the purpose of a plugin changelog?

Reply:
8ad72ba481